src/appleseed/foundation/image/colormap.h

```diff
@@ -93,6 +96,10 @@ class ColorMap
 
     Color3f evaluate_palette(float x) const;
 
+    APPLESEED_DLLSYMBOL std::array<float,4> compute_tile_highlight_color(
```

Comment:
That's not exactly how I saw it:

First, I would suggest to rename the function since it's not really specific to tile highlights. Maybe `evaluate_cosine_palette()`? There are many more of those by the way: https://www.shadertoy.com/view/ll2GD3, in the future we could consider adding more as needed.

Also, I think this should be a free function instead of static method as it's not related to the `ColorMap` class.

Finally, it would be more logical to return a `Color3f`.